<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task37_3Class</class>
 <widget class="QMainWindow" name="Task37_3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>745</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task37_3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>680</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>680</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>37</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string comment="1">1</string>
    </property>
    <property name="shortcut">
     <string comment="1"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>37</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>148</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>148</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>37</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>148</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>259</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>259</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>259</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>65</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>164</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>46</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Chanel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>206</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>121</width>
      <height>57</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Chanel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string extracomment="%">0%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string comment="--"/>
    </property>
    <property name="text">
     <string extracomment="-----"/>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>--</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Task37_3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>chanelPl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>chanelMi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>volPl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_13</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>volMi()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Task37_3Class</receiver>
   <slot>add1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add0()</slot>
  <slot>add1()</slot>
  <slot>add2()</slot>
  <slot>add3()</slot>
  <slot>add4()</slot>
  <slot>add5()</slot>
  <slot>add6()</slot>
  <slot>add7()</slot>
  <slot>add8()</slot>
  <slot>add9()</slot>
  <slot>chanelPl()</slot>
  <slot>chanelMi()</slot>
  <slot>volPl()</slot>
  <slot>volMi()</slot>
 </slots>
</ui>
